src/NServiceBus.Transport.Msmq/ReceiveStrategy.cs

```diff
@@ -101,6 +102,11 @@ protected void MovePoisonMessageToErrorQueue(Message message, MessageQueueTransa
 
         protected async Task<bool> TryProcessMessage(string messageId, Dictionary<string, string> headers, Stream bodyStream, TransportTransaction transaction)
         {
+            if(ttbrStrategy.DiscardDueToElapsedTtbr(headers))
```

Comment:
@mikeminutillo Does this truly means it is discarded? Meaning, the message will not be send to the audit queue?

Reply:
Correct. We never enter the pipeline so the message is never audited.